std_msgs/Header header

geometry_msgs/Point end_pos
geometry_msgs/Quaternion end_quat

geometry_msgs/Vector3 end_vel
geometry_msgs/Vector3 end_w